experimental/packages/otlp-exporter-base/test/browser/create-legacy-browser-delegate.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import { JsonLogsSerializer } from '@opentelemetry/otlp-transformer';
+import {
+  convertLegacyBrowserHttpOptions,
+  createLegacyOtlpBrowserExportDelegate,
+} from '../../src/index-browser-http';
+import {
+  createOtlpFetchExportDelegate,
+  createOtlpSendBeaconExportDelegate,
+  createOtlpXhrExportDelegate,
+} from '../../src/otlp-browser-http-export-delegate';
+import { OTLPExporterConfigBase } from '../../src';
+
+describe.only('createLegacyBrowserDelegate', function () {
```

Comment:
this currently disables all other tests for this package. 

```suggestion
describe('createLegacyBrowserDelegate', function () {
```